package.json

```diff
@@ -6,9 +6,11 @@
   "module": "./src/emel.js",
   "browser": "./dist/emel.umd.js",
   "type": "module",
+  "typings": "./typings.d.ts",
```

Comment:
Isn't this supposed to be `"types"`?

Reply:
I usually use `typings` for my libs, it can be referenced in TypeScript but both work

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package